### PR TITLE
Fix infoblox test

### PR DESCRIPTION
**Pull request by shubhamsg199**
Commands for disabling dns-infoblox and dhcp-infoblox are not properly documented. Hence adding  proper commands as per https://bugzilla.redhat.com/show_bug.cgi?id=2210256#c4

### CODE REVIEW

**Comment by shubhamsg199** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_infoblox.py -k test_infoblox_end_to_end

**Comment by shubhamsg199** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_infoblox.py -k test_infoblox_end_to_end

**Comment by shubhamsg199** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_infoblox.py -k test_infoblox_end_to_end

**Comment by shubhamsg199** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_infoblox.py -k test_infoblox_end_to_end

File: tests/foreman/destructive/test_infoblox.py [hunk 1]
```diff
@@ -232,27 +237,26 @@ def test_infoblox_end_to_end(
     # check hostname and ip is present in A record
     assert host.name in result.text
     assert host.ip in result.text
+    # delete host
+    module_target_sat.api.Subnet(id=subnet.id, domain=[]).update()
+    module_target_sat.api.Host(id=host.id).delete()
+    module_target_sat.api.Subnet(id=subnet.id).delete()
+    module_target_sat.api.Domain(id=domain.id).delete()
+    with pytest.raises(HTTPError):
+        host.read()
```
Comment (Gauravtalreja1) [inline tests/foreman/destructive/test_infoblox.py:246]: Shouldn't this be part of request.addfinalizer?

Comment (shubhamsg199) [reply tests/foreman/destructive/test_infoblox.py:246]: Not necessarily..we are also checking the deletion of the host. Also after disabling plugin the dhcp and dns feature is disabled so the host cannot be deleted.

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**